src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -12,6 +12,11 @@ export class AzureDeployPlugin {
   }
 
   private async deploy() {
+    this.serverless.cli.log("OPTIONS");
```

Comment:
Will Remove

Reply:
I'll assume you mean lines 15 & 16 here

Reply:
I was referring to all the log statements.  I put those in for debugging.